hivemind/utils/mpfuture.py

```diff
@@ -180,8 +180,10 @@ def _process_updates_in_background(cls, receiver_pipe: mp.connection.Connection)
                     future = future_ref()
 
                 if future is None:
-                    logger.debug(f"Ignoring update to future with uid={uid}: the future is already done or destroyed")
```

Comment:
This is one of the most frequent debug messages, however it is not very useful since ignoring the `MPFuture` result is considered okay in our code (for launching a background task). We agreed to remove it.